### DISCUSSION TITLE
cookbook finetune_muti_classfication.ipynb遇到的问题

### DISCUSSION THREAD

**Discussion by gaolegao2024** [zai-org/ChatGLM3#598] [Q&A] [answered]
ChatGLM3/cookbook/finetune_muti_classfication.ipynb
# 参数设置
lora_path = output_dir + "pytorch_model.bin"  //1.output_dir 这个文件目录字符串后面没有/ ，所以只会是 xxx/yyypytorch_model.bin形式，不对吧
2.生成出来的是pytorch_model.pt文件

if os.path.exists(lora_path)://所以这里是进不去的
    model.load_state_dict(torch.load(lora_path), strict=False)
	
	
但是当我修改成正确的路径后，预测的时候会报这个错：
RuntimeError: probability tensor contains either `inf`, `nan` or element < 0

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个大概率是访问了不应该访问了token id，你更新一下我们最新的hf配置文件
